### ISSUE TITLE
is there any plan to support cffi-1.14.0?

### ISSUE THREAD

**Issue by stevenxuwoss** [dveselov/python-libmagic#4]
uccessfully built python-libmagic
Failed to build cffi
ERROR: cryptography 2.8 has requirement cffi!=1.11.3,>=1.8, but you'll have cffi 1.7.0 which is incompatible.

**Comment by PrathamSoni** [comment]
Agree! Not sure if this repo is under active maintenance but loosening the cffi sub dependency would be great!!

**Comment by zfred68** [comment]
building '_cffi_backend' extension   ... fails
...
  c/call_python.c:235:63: error: invalid use of incomplete typedef ‘PyInterpreterState’ {aka ‘struct _is’}
      235 |         if (externpy->reserved1 != PyThreadState_GET()->interp->modules) {
          |                                                               ^~
    error: command 'gcc' failed with exit status 1

is there a work around? it been 2yrs since this issue been first reported

**Comment by Secret-chest** [comment]
still having issues
